cadc-access-control-server/src/main/java/ca/nrc/cadc/ac/server/oidc/AuthorizeAction.java

```diff
@@ -232,8 +233,13 @@ private void doOpenIDCodeFlow() throws Exception {
             // if authenticated (only possible by cookie) skip login form
             // formulate the authenticate redirect response
             
-            // TODO Alinga
-            // Add group check on rp.accessGroup here
+            // perform group check on rp.accessGroup 
+            if (!OIDCUtil.accessAllowed(clientID)) {
+                AuthorizeError authError = new AuthorizeError();
+                authError.error = "access group check failed";
```

Comment:
The same descriptive error message that is in LoginAction should be made here.

Reply:
Updated.